awesome.md

```diff
@@ -0,0 +1,31 @@
+# Awesome CRI-O [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
+
+This list follows the [awesome
+manifesto](https://github.com/sindresorhus/awesome/blob/master/awesome.md), and
+collects every useful resource about CRI-O.
```

Comment:
I am certain the list does not include *every* useful resource. Let's relax that a bit.

Reply:
s/every useful resource/useful resources/  Once we get to more than a dozen or so, we could add "many useful".

Reply:
Sounds good, I change it as suggested by @TomSweeneyRedHat.